src/Entity/Pia/ProcessingDataType.php

```diff
@@ -15,23 +15,23 @@
 use PiaApi\Entity\Pia\Traits\ResourceTrait;
 
 /**
- * @ORM\Entity
+ * @ORM\Entity(repositoryClass="PiaApi\Repository\ProcessingDataTypeRepository")
  * @ORM\Table(name="pia_processing_data_type")
  */
 class ProcessingDataType
 {
     use ResourceTrait;
 
     /**
-     * @ORM\Column(type="string")
+     * @ORM\Column(type="string", nullable=true)
```

Comment:
I would leave this field not nullable because a `ProcessingDataType` without any reference to the processing data type official list means nothing.

Reply:
I was based on the constructor which only takes a Processing object. I will change that.